Update common-issues.md (#1641)
codemagic-ci-cd/codemagic-docs: content/troubleshooting/common-issues.md

```

Please note that the ***XXXXXXXXX** refers to your **SSH-Key-ID**

> ### Github Integration - Repositories not showing up in the dropdown

Reviewer: ```suggestion
### GitHub Integration - Repositories not showing up in the dropdown
```

###### Description

> Github repostories won't show up in the dropdown while trying to add an app into Codemagic.

Reviewer: ```suggestion
GitHub repositories won't show up in the dropdown while trying to add an app to Codemagic.
```

###### Solution

> This will be solved by disconnecting and connecting to the github integration again.

Reviewer: ```suggestion
Try disconnecting the GitHub integration and then connecting it again.
```

> This will be solved by disconnecting and connecting to the github integration again.

Reviewer: ```suggestion
Try disconnecting the GitHub integration and then connecting it again.
```
